client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -65,6 +88,16 @@ class ActivityLogDay extends Component {
 			ts_end_site_day: tsEndOfSiteDay,
 			utc_date: moment.utc( tsEndOfSiteDay ).format( 'YYYY-MM-DD' ),
 		} );
+
+		this.setState( {
+			dayExpanded: true,
+		} );
+	};
+
+	handleCloseDay = () => {
+		this.setState( {
+			dayExpanded: false,
+		} );
 	};
```

Comment:
fun fact! we can get rid of some of the syntax noise

```js
handleCloseDay = () => this.setState( { dayExpanded: false } );
```

this is because all we're wanting to do is evaluate the expression. we don't care about the return value but it would be the return from `this.setState()` in this case because of how arrow functions work

Reply:
Addressed in ab73fd1a07